bigquery/indexer.py

```diff
@@ -38,6 +38,12 @@ def parse_args():
         type=str,
         help='Directory containing config files. Can be relative or absolute.',
         default=os.environ.get('DATASET_CONFIG_DIR'))
+    parser.add_argument(
+        '--billing_project_id',
+        type=str,
+        help='The ID of the GCP Project to bill, if separate from the project'
+        ' containing the BigQuery table(s)',
+        default=os.environ.get('BILLING_PROJECT_ID'))
```

Comment:
add required=True

Right now, for local run if I don't pass in BILLING_PROJECT_ID, I get:

indexer_1        |   File "/usr/local/lib/python2.7/site-packages/pandas_gbq/gbq.py", line 819, in read_gbq
indexer_1        |     raise TypeError("Missing required parameter: project_id")
indexer_1        | TypeError: Missing required parameter: project_id
bigquery_indexer_1 exited with code 1

Adding required=True will give a friendly error message.

Reply:
Done